src/app/pluginAdapter.js

```diff
@@ -1,16 +1,18 @@
 // @flow
 
-import type {Graph, NodeAddressT} from "../core/graph";
+import type {Graph, NodeAddressT, EdgeAddressT} from "../core/graph";
 
 export interface Renderer {
   nodeDescription(NodeAddressT): string;
+  edgeVerb(EdgeAddressT, "FORWARD" | "BACKWARD"): string;
```

Comment:
There are a number of alternate factorizations that make sense here.
First, this function could return `{forward: string, backward: string}`
instead of requiring two invocations (and two `try`-`catch`es, etc.).
Second, it would seem reasonable to me to move these onto an
`edgeTypes()` method of the plugin adapter itself, for symmetry with the
node adapter. This seems like a low-difficulty refactor, so I’m okay
with merging the code as is.